### PR TITLE
Al/v2.12.1 fix issue with awssdk.security token

### PR DESCRIPTION
**Pull request by alouvion-aneo**

### CODE REVIEW

File: armonik-versions.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-core=0.12.2
-worker=0.9.2
+core=0.12.3
+worker=0.9.4
 admin-gui=0.9.0
-samples=v2.12.0
+samples=v2.12.1
```
Comment (lemaitre-aneo) [inline armonik-versions.txt:4]: Is there such a version?

**Review by lemaitre-aneo — COMMENTED** [review]
Could we have an item in the CHANGELOG?

File: versions.tfvars.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "armonik_versions": {
     "infra":     "2.12.0",
```
Comment (lzianekhodja) [inline versions.tfvars.json:3]:
```suggestion
    "infra":     "2.12.1",
```

File: versions.tfvars.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-    "core":      "0.12.2",
+    "core":      "0.12.3",
     "api":       "3.6.0",
     "gui":       "0.9.0",
-    "extcsharp": "0.9.2",
+    "extcsharp": "0.9.4",
     "samples":   "2.12.0"
```
Comment (lzianekhodja) [inline versions.tfvars.json:8]:
```suggestion
    "samples":   "2.12.1"
```

**Comment by alouvion-aneo** [comment]
I will do it in another PR

**Pull request closed without merging**